controlpanel/api/cluster.py

```diff
@@ -346,3 +356,120 @@ def get_repository(user, repo_name):
         return github.get_repo(repo_name)
     except GithubException.UnknownObjectException:
         return None
+
+
+class ToolDeploymentError(Exception):
+    pass
+
+
+def deploy_tool(tool, user, **kwargs):
+    values = {
+        'username': user.username.lower(),
+        'Username': user.username.lower(),  # XXX backwards compatibility
```

Comment:
I know the comment says "backward compatibility" but why do we pass both?

Oh is it because some of the tools have `Username` as value instead of `username`?

Reply:
Yes, exactly that. I thought it was easier to just pass both variants and standardise the Helm charts later.